nin/dasBoot/BEATBEAN.js

```diff
@@ -5,8 +5,9 @@
     BEAN = 0;
     BEAT = false;
 
-    var subdivision = 6;
-    var beatsPerMinute = 110 * subdivision;
+    var musicInfo = PROJECT.get('music', {});
+    var subdivision = musicInfo.subdivision || 6;
+    var beatsPerMinute = (musicInfo.bpm || 110) * subdivision;
```

Comment:
it is more BEATs per minute, since it is the number of beatbean BEATs, e.g. the thing counted by BEAN.